### PR TITLE
Adding copy-to-clipboard functionality

### PR DESCRIPTION
**Pull request by nelliemckesson**
This is a revised version of #2131 (which got corrupted by some weird git stuff). Adds the copy-to-clipboard button to code blocks, including some tooltip text that only shows up when you directly hover over the button. @lurch @aallan go ahead and test the latest code!

### CODE REVIEW

**Review by aallan — APPROVED** [review]
LGTM

**Comment by lurch** [comment]
@nelliemckesson Is it easy to tweak the styling on this, so that you get a "hand" mouse-cursor when hovering over the Copy-to-Clipboard button, rather than just displaying the default "arrow" mouse-cursor?
@aallan Assuming of course that this would be desirable :wink:  (IMHO the change of mouse-cursor would make it more obvious that something is "clickable")

Only a minor bugette (so perhaps not worth fixing), but if I click the Copy-to-Clipboard button and then quickly move the mouse cursor out of the code-box and then back over the Copy-to-Clipboard button, it _briefly_ displays "Copied! Copy to Clipboard", and then reverts to saying just "Copy to Clipboard".

**Review by gk351 — COMMENTED** [review]
12321@lbt

File: makefiles/jekyll.mk [hunk 1]
```diff
@@ -28,7 +28,7 @@ $(OUTPUT_DIR):
 $(OUTPUT_DIR)/%: | $(JEKYLL_ASSETS_DIR)/% $(OUTPUT_DIR)
 	cp -r $(firstword $|) $@
 
-JEKYLL_DIRS = $(OUTPUT_DIR)/_data $(OUTPUT_DIR)/_layouts $(OUTPUT_DIR)/_includes $(OUTPUT_DIR)/css $(OUTPUT_DIR)/scripts
+JEKYLL_DIRS = $(OUTPUT_DIR)/_data $(OUTPUT_DIR)/_layouts $(OUTPUT_DIR)/_includes $(OUTPUT_DIR)/_plugins $(OUTPUT_DIR)/css $(OUTPUT_DIR)/scripts
```
Comment (gk351) [inline makefiles/jekyll.mk:32]: 9a8db46c952cd933eed7907978d24b3a7bfefdd5

Comment (gk351) [inline makefiles/jekyll.mk:32]: readme.txt

**Review by gk351 — COMMENTED** [review]
但

**Pull request merged**